donor-portal/src/pages/donate-now/styles.css

```diff
@@ -27,14 +27,18 @@
   margin: 2rem 0;
 }
 
+.package-image {
+  width: 30%;
+}
+
 .package-card {
   display: flex;
   margin-bottom: 2rem;
   border: 1px solid var(--elixir-color-gray-500);
 }
 
 .card-details {
-  width: 100%;
+  width: 70%;
```

Comment:
This should not be from percentages. Provide image size from pixels and let `card-details` take the rest of the space. 

Use a `flex`.

Reply:
sure